### DISCUSSION TITLE
How to parse polymorphic JSON using converters?

### DISCUSSION THREAD

**Discussion by CypherpunkSamurai** [square/retrofit#3770] [Q&A]
Hello Everyone,

I'm trying to use retrofit for an api endpoint that returns two different json for two different cases.
```json
{
    "status": "working",
    "result": "hmm"
}
```

```json
{
    "error": "code here"
}
```

How can I make retrofit work for both the cases?

**Comment by gold24park** [comment]
I recommend this way:
Receive the response with Map<String, Any> or Any first, then convert it.
